cli/cmd/version.go

```diff
@@ -39,7 +43,24 @@ var availableVersionCmd = &cobra.Command{
 			return err
 		}
 
-		fmt.Fprint(cmd.OutOrStdout(), v)
+		fmt.Fprintln(cmd.OutOrStdout(), v)
+		return nil
+	},
+}
+
+var lockHappyVersionCmd = &cobra.Command{
+	Use:          "lock",
+	Short:        "Create a .happy/version.lock file",
+	Long:         "Create a .happy/version.lock file in project root to specify which version of Happy should be used with this project. This will overwrite any existing version.lock file.",
+	SilenceUsage: true,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		versionFile, version, err := CreateHappyVersionFile(cmd)
+		if err != nil {
+			fmt.Fprint(cmd.Parent().ErrOrStderr(), err)
```

Comment:
```suggestion
			log.Debug(cmd.Parent().ErrOrStderr(), err)
```

unless this was just for testing, in which case ignore this comment and remove before its finished